### PR TITLE
Update footer and Twitter desc meta tag

### PR DESCRIPTION
**Pull request by connorchan**

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
   <div>
     <div class="main">
       <div class="logo"><img src="{{ "/images/logo.svg" | relative_url }}" /></div>
-      <span class="md-hidden">The open source multicloud control plane</span>
```
Comment (jbw976) [inline _includes/footer.html]: should this be changed to "Manage your applications and infrastructure the Kubernetes way" instead of being removed?

Comment (connorchan) [reply _includes/footer.html]: @jbw976 per Matt, we are simply removing this text from the footer

**Comment by connorchan** [comment]
@grantgumina yes, we removed that in an earlier PR this morning

**Comment by grantgumina** [comment]
Same with the copy changes and typo fixes for "the source of truth" section under "The Kubernetes Way"?

**Comment by grantgumina** [comment]
Kk - looking at the other PRs from this morning. Looks great.

**Pull request merged**